### DISCUSSION TITLE
defaultNoteValue Used as Default for Velocity

### DISCUSSION THREAD

**Discussion by LiamGallagherBC** [Tracktion/tracktion_engine#181] [General]
This is a two part query. 
The first part, is being a zero-th time contributor, I was wondering if feedback about variable name choice is within the scope of considerations for contributions to the repository. What's the etiquette here?

The second part is that as seen in tracktion_StepClip.h's enum Defaults, and then used in tracktion_StepClip.cpp (and else where), defaultNoteValue is used as the default velocity value. I should think that defaultNoteVelocity is a more appropriate identifier, more so since note value and note number are often used interchangeably when talking about MIDI.

Let me know if these types of contributions are welcome, and I can put a PR together. If there's a contribution guide please let me know so I can make my PR fit, I wasn't able to find any mention of that in the ReadMe.

Thanks,
Liam Gallagher

**Comment by drowaudio** [comment]
The honest answer is "it depends". Occasionally poor variable names slip under the code review radar and it's fine to update them. But we usually look at the impact that change would have before doing so.

In this case, it's probably a good idea to update is to `defaultNoteVelocity` for consistency elsewhere and it's unlikely to be used much by people as it's an internal default for `noteValue`. The `juce::CachedValue<int> noteValue` is also a poor choice of name and should probably be `noteVelocity` but this one is tricker as people almost certainly will be relying on it.
